linera-execution/src/evm/revm.rs

```diff
@@ -51,6 +51,9 @@ const PROCESS_STREAMS_SELECTOR: &[u8] = &[227, 9, 189, 153];
 /// only when creating a new instance of a shared contract
 const INSTANTIATE_SELECTOR: &[u8] = &[156, 163, 60, 158];
 
+/// Returns the deployed bytecode of the contract.
```

Comment:
Can you expain more in-depth why the sequence of these bytes is valid for getting the deployed bytecode?